j2lint/rules/jinja_variable_name_format_rule.py

```diff
@@ -34,4 +35,8 @@ def check(self, file, line):
             if matches:
                 print(matches)
```

Comment:
Hmm I know you did not change this but wondering what the print statement is doing here

Reply:
yes,we can remove this